### ISSUE TITLE
node-gyp Fail When Installing from npm or source

### ISSUE THREAD

**Issue by KICKGodz** [stackgl/headless-gl#270]
I'm currently running a Ubuntu v20.04 ARM64 on Oracle and keep getting an install error when trying to download. 

I tried downloading through FEX to see if that would change anything but still the same.

Here is my error:
```bash
npm ERR! code 1
npm ERR! path /home/ubuntu/projects/tiktok/client/node_modules/gl
npm ERR! command failed
npm ERR! command sh -c prebuild-install || node-gyp rebuild
npm ERR! prebuild-install warn install No prebuilt binaries found (target=12.22.9 runtime=node arch=arm64 libc= platform=linux)
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.4.1
npm ERR! gyp info using node@12.22.9 | linux | arm64
npm ERR! gyp info find Python using Python version 3.10.12 found at "/usr/bin/python3"
npm ERR! gyp http GET https://nodejs.org/download/release/v12.22.9/node-v12.22.9-headers.tar.gz
npm ERR! gyp http 200 https://nodejs.org/download/release/v12.22.9/node-v12.22.9-headers.tar.gz
npm ERR! gyp http GET https://nodejs.org/download/release/v12.22.9/SHASUMS256.txt
npm ERR! gyp http 200 https://nodejs.org/download/release/v12.22.9/SHASUMS256.txt
npm ERR! gyp info spawn /usr/bin/python3
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/home/ubuntu/projects/tiktok/client/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/ubuntu/projects/tiktok/client/node_modules/gl/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/ubuntu/projects/tiktok/client/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/home/ubuntu/.cache/node-gyp/12.22.9/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/home/ubuntu/.cache/node-gyp/12.22.9',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/home/ubuntu/projects/tiktok/client/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/home/ubuntu/.cache/node-gyp/12.22.9/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/home/ubuntu/projects/tiktok/client/node_modules/gl',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! /bin/sh: 1: python: not found
npm ERR! gyp: Call to 'python commit_id.py check ..' returned exit status 127 while in angle/src/angle.gyp. while loading dependencies of binding.gyp while trying to load binding.gyp
npm ERR! gyp ERR! configure error 
npm ERR! gyp ERR! stack Error: `gyp` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onCpExit (/home/ubuntu/projects/tiktok/client/node_modules/node-gyp/lib/configure.js:325:16)
npm ERR! gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
npm ERR! gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
npm ERR! gyp ERR! System Linux 5.15.0-1047-oracle
npm ERR! gyp ERR! command "/usr/bin/node" "/home/ubuntu/projects/tiktok/client/node_modules/.bin/node-gyp" "rebuild"
npm ERR! gyp ERR! cwd /home/ubuntu/projects/tiktok/client/node_modules/gl
npm ERR! gyp ERR! node -v v12.22.9
npm ERR! gyp ERR! node-gyp -v v9.4.1
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ubuntu/.npm/_logs/2023-12-22T20_34_01_420Z-debug-0.log
```

Here is the full debug file:
[debug.log](https://github.com/stackgl/headless-gl/files/13756601/debug.log)

I've tried installing using Node versions 20, 18, 16, and 14 and it still hasn't worked.
Is this because of ARM or another reason? I installed all packages required and also those that node-gyp requires.

**Comment by zFernand0** [comment]
You may want to doble check your dependencies.
For instance, you are missing python. In my case I had to make sure the `python-is-python3` (or `2to3`) was installed.
And also decided to `/bin/sh -> bash` (instead of the default `dash`)

I'm also trying to install GL on arm64 and hitting something related to the GCC issue below:
- #271

**Comment by linev** [comment]
We suffer from this problem for more than one year.
Finally I just fork repository and fix this annoying problem there.

https://github.com/linev/headless-gl

https://www.npmjs.com/package/jsroot-gl

I hope that such simple fix also can be applied here as well.

**Comment by dhritzkiv** [comment]
Prebuilt binaries are available in the latest release (which also fixes builds from source): [v8.0.2](https://github.com/stackgl/headless-gl/releases/tag/v8.0.2)

**Comment by monteslu** [comment]
> Prebuilt binaries are available in the latest release (which also fixes builds from source): [v8.0.2](https://github.com/stackgl/headless-gl/releases/tag/v8.0.2)

Is arm64 linux supported?  only seeing arm x86 in the released
